tests/lib/validator_test.py

```diff
@@ -652,13 +653,100 @@ def test_mac_restriction_in_current_mac_in_desire(self):
                 original_desired_state, current_state
             )
 
-    def _gen_iface_state(self, bond_name):
-        return {
-            Interface.NAME: bond_name,
-            Interface.TYPE: InterfaceType.BOND,
+
+class TestPortsValidator:
+
+    ipv_up = pytest.mark.parametrize("ipv_up", [(True, False), (False, True)])
```

Comment:
```python
parametrize_ip_ver = pytest.mark.parametrize(
    "ip_state",
    [(True, False), (False, True)],
    ids=["ipv4 on, ipv6 off", "ipv4 off, ipv6 on"],
)
```

Reply:
Fixed this!! Thanks a lot for all your reviews!